dashboard/src/app/stacks/stack-details/stack.controller.ts

```diff
@@ -16,12 +16,31 @@
  * @author Ann Shumilova
  */
 export class StackController {
+  loading: boolean;
+  isLoading: boolean;
+  isCreation: boolean;
+
+  stackId: string;
+  stackName: string;
+  stackContent: string;
+  invalidStack: string;
+
+  $log;
+  stack;
+  $filter;
+  cheStack;
+  $timeout;
+  $location;
+  $mdDialog;
```

Comment:
for information, you may use https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/angular-material/angular-material.d.ts#L114 type
as we have angular-material definitions there https://github.com/eclipse/che/blob/master/dashboard/tsd.json#L26

the same for Angular object, you can use TypeScript definition (which are not any)